src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Extensions/OpenApiParameterAttributeExtensions.cs

```diff
@@ -20,8 +23,9 @@ public static class OpenApiParameterAttributeExtensions
         /// <param name="attribute"><see cref="OpenApiParameterAttribute"/> instance.</param>
         /// <param name="namingStrategy"><see cref="NamingStrategy"/> instance.</param>
         /// <param name="collection"><see cref="VisitorCollection"/> instance.</param>
+        /// <param name="version">OpenAPI spec version.</param>
```

Comment:
This line should be `/// <param name="version"><see cref="OpenApiVersionType"/> value.</param>`

Reply:
I fixed it. Thank you so much for your thorough review. 😉 Additionally, the `dateTimeOffset` type is also specified.